tests/phpunit/tests/link/postNavigationLinkAttributeFilters.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+/**
+ * Test post navigation link attribute filters.
+ *
+ * @since 6.2.0
+ * @ticket 55751
```

Comment:
```suggestion
 * Tests post navigation link attribute filters.
 *
 * @since 6.2.0
```
- Change `Test` to `Tests`.
- Remove `@ticket` annotation at the class level. It's only needed for `test_` methods.